tests/link/pytorch/test_basic.py

```diff
@@ -235,3 +235,20 @@ def test_arange():
         FunctionGraph([start, stop, step], [out]),
         [np.array(1), np.array(10), np.array(2)],
     )
+
+
+def test_eye():
+    N = scalar("N", dtype="int64")
+    M = scalar("M", dtype="int64")
+    k = scalar("k", dtype="int64")
+
+    out = eye(N, M, k, dtype="float32")
+
+    trange = range(1, 6)
+
+    fn = function([N, M, k], out, mode=pytorch_mode)
```

Comment:
is `pytorch_mode` defined somewhere else?

Reply:
Yep, globally on the same file.